ark-cli/src/main.rs

```diff
@@ -5,6 +5,9 @@ use std::str::FromStr;
 
 use data_pdf::{render_preview_page, PDFQuality};
 use data_resource::ResourceId;
+use data_resource::ResourceIdTrait;
+pub(crate) type HashType = <ResourceId as ResourceIdTrait>::HashType;
```

Comment:
We should provide more descriptive name to the end-user. Ideally, `ResourceId`.

By the way, if we make `HashType` a parameter of common `ResourceId` structure, we could avoid fixing the name for it (see the comment about generic `ResourceId` structure).

Reply:
> We should provide more descriptive name to the end-user. Ideally, `ResourceId`.

Great point. totally agree. 
Renamed `HashType` to `ResourceId`. This shifts the long use statement from other crates to `data-resource`